### DISCUSSION TITLE
Would it be reasonable to aim for VSCodium Coding Packages?

### DISCUSSION THREAD

**Discussion by GitMensch** [VSCodium/vscodium#1470] [Ideas]
With the October 2020 update MS highlights that people starting coding likely want to use their coding packs, currently https://code.visualstudio.com/learntocode shows a Java and a Python one (at least it is plain honest "By using VS Code, you agree to its license and privacy statement.").

The question is: Would it be reasonable to aim for something similar with VSCodium?
Maybe a community-created kind of guideline "how to create your own coding pack"?

**Comment by daiyam** [comment]
Let's keep it active.
